at_response.c

```diff
@@ -1963,9 +1965,25 @@ int at_response (struct pvt* pvt, const struct iovec iov[2], int iovcnt, at_res_
 							return at_response_cgmi (pvt, str);
 
 						case CMD_AT_CGMM:
-							ast_debug (1, "[%s] Got AT_CGMM data (model info)\n", PVT_ID(pvt));
-							return at_response_cgmm (pvt, str);
-
+							{
+								ast_debug (1, "[%s] Got AT_CGMM data (model info)\n", PVT_ID(pvt));
+								int ret = at_response_cgmm (pvt, str);
+
+								if (0==strncmp(pvt->model, "E173", 4)) {
```

Comment:
Yes. That's also a common pattern, but not used here.

Please do `... == 0)` (with spaces).

Reply:
Also, this is rather kludgy to hardcode a model in the middle of things. If you could make that a little more dynamic (even if it's just a hardcoded list elsewhere), it'd be nice.